test/unit/tech/html5.test.js

```diff
@@ -442,3 +444,30 @@ test('Html5#reset calls Html5.resetMediaElement when called', function() {
 
   Html5.resetMediaElement = oldResetMedia;
 });
+
+test('Html5#setSrc clears currentSource_ after loadstart', function() {
+
+  let disposed = false;
+  let thing = {
+    off: () => {},
+    one: (el, type, fun) => {
+      el.one(type, Fn.bind(thing, fun));
+    },
```

Comment:
It does successfully test that setSrc clears contentSource_ after loadstart. Do you have any suggestions on how to make this more real?